src/components/CommentsPage.jsx

```diff
@@ -0,0 +1,19 @@
+import { useEffect, useState } from "react"
+import { fetchCommentsByArticleId } from "./api"
+
+function CommentsPage(article_id) {
```

Comment:
This is not relevant to this ticket, so should be removed and submitted as part of the relevant PR when the time comes please